docs/guides/using-slack-cli-with-bolt-frameworks.md

```diff
@@ -26,18 +26,18 @@ View more samples
 
 You will then be prompted to choose between **Bolt for JavaScript** or **Bolt for Python**. Choose your favorite flavor.
 
-Your app will be cloned from the respective [JavaScript](https://github.com/slack-samples/bolt-js-starter-template) or [Python](https://github.com/slack-samples/bolt-python-starter-template) project template on our Slack Platform Sample Code repository, and its project dependencies will be installed. Then, `cd` into your project folder. 
+Your app will be cloned from the respective [JavaScript](https://github.com/slack-samples/bolt-js-starter-template) or [Python](https://github.com/slack-samples/bolt-python-starter-template) project template on our Slack Platform Sample Code repository, and its project dependencies will be installed. Then, `cd` into your project folder.
 
 :::info
 
 For Bolt for Python projects, automatic project dependency installation is currently unsupported, and will need to be done manually. For more information, refer to [Getting started with Bolt for Python](https://tools.slack.dev/bolt-python/getting-started).
 
 :::
 
-To run your new app, use the `slack run` command with the experiment flag as follows:
+To run your new app, use the `slack run` command as follows:
```

Comment:
note: I think this was left over from when we had `--experiment bolt`.

Reply:
Thanks for reviewing these pages as part of this change 👾 ✨